### DISCUSSION TITLE
美國限制企業向中國出售先進晶片技術及設備

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#476] [General]
這真的有點狠，要向中國出先進的晶片技術或設備，皆需要得到特別許可證。這對中美兩方會有哪些影響呢？

https://cn.nytimes.com/usa/20221008/biden-chip-technology/zh-hant/

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
感覺這不只對中國國內產業有影響
對美國本土的晶片輸出也會有一定程度的打擊

**Comment by williamyen0623** [comment]
這無疑對雙方，甚至是其他從事相關產業的國家都會有影響
雖然會對總體經濟造成不好的影響，但中國偷技術、違反學術倫理等等行為真的是讓人無法忍受

**Comment by NcuJerryKao** [comment]
對兩岸的影響

**Comment by Ben6uo** [comment]
這樣很大限制了中國的發展 對臺灣來說還不錯?

**Comment by limitx0** [comment]
雖然限制了中國的發展，但是感覺對美國經濟也有不少影響

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
這無疑對兩國的經濟都會受到損失

**Comment by awehao** [comment]
雙輸局面，我覺得某種程度也打壓了自家的企業

**Comment by larry40305** [comment]
傷敵1000，自損800

**Comment by lance111l** [comment]
有竞争 就有打压     原来美国和西方打压中国少吗   现在的打压只是有短期效果

**Comment by yjpxi** [comment]
我覺得全球半導體供應鏈可能會因為這一政策而重組，更多的企業可能會將生產轉移到其他國家，以避免受到美中競爭的影響
